src/content/english.js

```diff
@@ -218,44 +218,50 @@ const TESTIMONIALS = [
         name: 'Oscar Lafarga',
         quote: 'SetLife has become my long-term commitment to a sustainable future when it comes to technology and its impact on society. In a world where information & communication moves at the speed of light, constraining our education to the classroom leaves us woefully under-equipped to deal with the major global problems we face. My goal is to incentivize teaching as one of the highest-paid and most valued professions, toppling the knowledge-hoarding power structures that sustain a global inequality of wealth',
         date: 'President & CTO',
-        portrait: 'https://user-images.githubusercontent.com/4914611/63651576-6e1cb200-c756-11e9-88f2-87cb80faa5a1.png'
+        portrait: 'https://user-images.githubusercontent.com/49292858/68433886-0808ce00-018e-11ea-9818-64cd4372bdc6.png'
+    },
+    {
```

Comment:
Let's sort these by putting those with testimonials first so Oscar, Viki, Quinn, Sofia, followed by the rest